package.json

```diff
@@ -31,9 +32,11 @@
     "npm-run-all": "^4.0.2",
     "prettier": "^1.4.4",
     "proxyquire": "^1.8.0",
+    "serial-mocha": "0.0.4",
```

Comment:
Same here... Do we use this module somehow?
I was trying to read [**serial-mocha**](https://www.npmjs.com/package/serial-mocha) docs, but it doesn't look like we're using the `serial-mocha` CLI script, so wasn't sure if this was used.